guides/source/active_job_basics.md

```diff
@@ -330,6 +330,31 @@ class GuestsCleanupJob < ActiveJob::Base
 end
 ```
 
+### Deserialization
+
+GlobalID, which Active Job uses, allows serializing full ActiveRecord objects as an argument to `#perform`.
```

Comment:
I guess you can remove the "which Active Job uses", it is already explained earlier in the guide. Also if you are not speaking about the constant it is actually "Active Record". Moreover, you could replace "as an argument" with "passed".